pallets/collective/src/tests.rs

```diff
@@ -1118,7 +1118,7 @@ fn motions_all_first_vote_free_works() {
         assert_eq!(close_rval.unwrap().pays_fee, Pays::No);
 
         // trying to close the proposal, which is already closed.
-        // Expecting error "ProposalMissing" with Pays::Yes
+        // Expecting error "ProposalNotExists" with Pays::Yes
```

Comment:
Nothing is missing. the unit test verify two items. 
1. the extrinsic is failed, because the proposal already closed.
2. to check the result if the call is Pays::Yes or Pays::No. We expect Pays::Yes, and compare it via assert_eq!

Reply:
I see. Then I suggest the below:

Error: "ProposalNotExists" with Pays::Yes. Trying to close a proposal that is already closed.